packages/yourloops/components/patient-data/patient-data.utils.ts

```diff
@@ -161,24 +161,46 @@ export class PatientDataUtils {
     return patientData
   }
 
-  private async fetchPatientDataRanges(dateRange: DateRange): Promise<PatientData> {
+  private async fetchPatientDataRanges(dateRange: DateRange): Promise<MedicalData> {
     const ranges = this.partialDataLoad.getMissingRanges(dateRange)
+    const aggregatedData: MedicalData = {
+      basal: [],
+      bolus: [],
+      alarmEvents: [],
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏Almost the alphabetical order... :sweat_smile:

Reply:
ok